packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -113,3 +116,163 @@ class DefaultTaggedText extends StatelessWidget {
+
+  /// The child that will be laid out last along [axis].
+  final Widget secondChild;
+
+  /// The fraction of the layout to allocate to [firstChild].
+  ///
+  /// [secondChild] will receive a fraction of `1 - initialFirstFraction`.
+  final double initialFirstFraction;
+
+  /// The key passed to the divider between [firstChild] and [secondChild].
+  ///
+  /// Visible to grab it in tests.
+  @visibleForTesting
+  Key get dividerKey => Key('$this dividerKey');
+
+  /// The size of the divider between [firstChild] and [secondChild] in
+  /// logical pixels (dp, not px).
+  static const double dividerMainAxisSize = 10.0;
+
+  @override
+  State<StatefulWidget> createState() => _SplitState();
+}
+
+class _SplitState extends State<Split> {
+  double firstFraction;
+  double get secondFraction => 1 - firstFraction;
+  bool get isHorizontal => widget.axis == Axis.horizontal;
+
+  @override
+  void initState() {
+    super.initState();
+    firstFraction = widget.initialFirstFraction;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(builder: _buildLayout);
+  }
+
+  Widget _buildLayout(BuildContext context, BoxConstraints constraints) {
+    final width = constraints.maxWidth;
+    final height = constraints.maxHeight;
+    final axisSize = isHorizontal ? width : height;
+    const halfDivider = Split.dividerMainAxisSize / 2.0;
+    // The fraction of the layout the divider needs to take up from each child.
+    final halfDividerFraction = halfDivider / axisSize;
+
+    final sanitizedFirstFraction =
+        firstFraction.clamp(halfDividerFraction, 1.0 - halfDividerFraction);
+    final sanitizedSecondFraction =
+        secondFraction.clamp(halfDividerFraction, 1.0 - halfDividerFraction);
+
+    final firstSize = axisSize * sanitizedFirstFraction - halfDivider;
+    final secondSize = axisSize * sanitizedSecondFraction - halfDivider;
+
+    void updateSpacing(DragUpdateDetails dragDetails) {
+      final delta = isHorizontal ? dragDetails.delta.dx : dragDetails.delta.dy;
+      final fractionalDelta = delta / axisSize;
+      setState(() {
+        // Update the fraction of space consumed by the children,
+        // being sure not to allocate any of them negative space.
+        firstFraction += fractionalDelta;
+        firstFraction = firstFraction.clamp(0.0, 1.0);
+      });
+    }
+
+    final children = [
+      SizedBox(
+        width: isHorizontal ? firstSize : width,
+        height: isHorizontal ? height : firstSize,
+        child: widget.firstChild,
+      ),
+      GestureDetector(
+        key: widget.dividerKey,
+        onHorizontalDragUpdate: isHorizontal ? updateSpacing : null,
+        onVerticalDragUpdate: isHorizontal ? null : updateSpacing,
+        child: SizedBox(
+          width: isHorizontal ? Split.dividerMainAxisSize : width,
+          height: isHorizontal ? height : Split.dividerMainAxisSize,
+          child: const Center(
+            child: Text(
+              ':::::::',
```

Comment:
use the existing png for this instead of using text.
Alternately, find a more material design appropriate splitter icon.

Reply:
Naturally we aren't going to land that :D

Do you know the name of the png file for the splitter grip?

Reply:
The splitter icons are currently stored as data urls in the app as that is the typical way they are included as the images are tiny.
You can also find the pngs here.
https://github.com/nathancahill/split/tree/master/packages/splitjs/grips
```

.gutter.gutter-horizontal {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
    cursor: col-resize;
    height: 100%;
}

.gutter.gutter-vertical {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAFAQMAAABo7865AAAABlBMVEVHcEzMzMzyAv2sAAAAAXRSTlMAQObYZgAAABBJREFUeF5jOAMEEAIEEFwAn3kMwcB6I2AAAAAASUVORK5CYII=');
    cursor: row-resize;
    width: 100%;
}

```